src/main.ts

```diff
@@ -12,3 +15,9 @@ new Vue({
   store: new Vuex.Store({}),
   render: (h) => h(App),
 }).$mount('#app')
+
+// required inline due to front-loading of imports and having jQuery dependency
+import('jquery')
+  .then(() => {
+    import('bootstrap')
```

Comment:
Imports using this syntax are `async`/non-blocking. Do we want to `await` on these?

Reply:
I'm tried this but I was getting:

`Module parse failed: Cannot use keyword 'await' outside an async function`

Any tips? I've tried:

```
await import('jquery')
  .then(() => {
    await import('bootstrap')
  })
```

And 

```
await import('jquery')
  .then(() => {
    import('bootstrap')
  })
```

Reply:
Discussed at Friday catch up - without await is fine for now.